content/blog/250819-redefining-cng-ccrp.md

```diff
@@ -0,0 +1,416 @@
+that defines:
+
+* A RESTful API for listing, fetching metadata for, and querying datasets
+* A small but flexible query syntax for both array and tabular data
+* A two-phase protocol enabling parallel downloads
+* Conformance classes for incremental implementation
+
+But this is just the beginning. For CCRP to succeed, it needs help from a
+number of stakeholders.
+
+**Questions for users**
+
+* Use case validation: Does this work with your access patterns?
+* Data format and tool integrations: Is this effort targeting the right formats
+  and tools?
+
+**Questions for data producers**
+
+* Chunking strategy examples: How would smaller chunks change your data?
+* Access pattern documentation: What queries do your users need?
+* Performance requirements: What latency is acceptable?
+
+**Questions for cloud providers**
+
+* Technical feasibility review: What would native implementation require?
+* Performance projections: How much better could this be than a proxy?
+* Integration possibilities: How would this fit with existing services?
+
+**Other areas for community engagement**
+
+* Specification feedback: What works? What doesn’t? What's missing?
+* Proof-of-concept implementations: Can we demonstrate the concepts,
+  performance gains, and quality of life improvements?
+* Tooling integrations: Where do we need to add support for CCRP?
+
+### An invitation
+
+Again, **the [tyranny of the
+chunk](https://element84.com/software-engineering/chunks-and-chunkability-tyranny-of-the-chunk/)
+doesn't have to be permanent.** CCRP is an effort to fight the tyrrany of the chunk.
+
+Ideally this work is owned and driven forward by the community, not just me.
+I’m just trying to kickstart it with some examples of what I’ve been thinking.
+Everything is on the table, it’s all up for discussion. The spec certainly has
+gaps, problems, and areas needing refinement. The examples are made up,
+admittedly not great, and need to be revised with real values for both raster
+and tabular datasets.  Actual proof-of-concept implementations are definitely
+needed to better identify what works and what doesn’t, and to allow those
+learnings to feedback into and inform the spec.
+
+So again, if you work with large chunked datasets, I want to hear from you. If
+you're building data infrastructure, I want your technical feedback. If you
+work at a cloud provider, let's talk about what native implementation would
+take. Or maybe you think this idea is flawed and can never work: awesome\!
+Perhaps you hate the name: great\! I’d love to understand any and all concerns,
+or, better, to know what you’d propose instead.
+
+So take a look at [the documentation and
+specification](https://ccrp-dev.github.io/ccrp). The [github
+repo](https://github.com/ccrp-dev/ccrp) is a great place to open issues, ask
+questions, or propose edits via PRs. We also have [a short
+form](https://docs.google.com/forms/d/e/1FAIpQLSdV--Hl86XwBhMDSfNDBXDST4ZEBrZSXQ7hfuIl28NJWVOZag/viewform?usp=dialog)
```

Comment:
Just wanted to check to make sure using a google form is okay here. I know CNG has had a specific form page made in the past (at least for the CNG Conf 2026 interest form).